### PR TITLE
Adds a delete verb to the vv_drop_down

### PR DESCRIPTION
**Pull request by Cyberboss**
Requires +DEBUG. You can already do this by marking the object and Sdql'ing qdel on it

### CODE REVIEW

**Comment by Iamgoofball** [comment]
cant you just right click delete tho

**Comment by Cyberboss** [comment]
Not datums

**Comment by Iamgoofball** [comment]
Ah

**Comment by optimumtact** [comment]
feature freeze

**Comment by PKPenguin321** [comment]
could really use this though please dont forget once the freeze ends

**Comment by lzimann** [comment]
conflicts

File: code/datums/datumvars.dm [hunk 1]
```diff
@@ -27,6 +27,7 @@
 	. += "---"
 	.["Call Proc"] = "?_src_=vars;proc_call=\ref[src]"
 	.["Mark Object"] = "?_src_=vars;mark_object=\ref[src]"
+	.["Delete"] = "?_src_vars;delete=\ref[src]"
```
Comment (lzimann) [inline code/datums/datumvars.dm]: `_src_=vars` and not `_src_vars`

**Pull request merged**